src/ansys/fluent/core/services/field_data.py

```diff
@@ -838,6 +838,121 @@ def _extract_field(field_datatype, field_size, chunk_iterator):
     return fields_data
 
 
+class _FieldDataBase:
```

Comment:
@prmukherj , This is actually public - it contributes to the public interfaces of the derived class - so we should give it a public name. I would suggest `BaseFieldData` because it reads better.

Reply:
ok, sure. Thank you

Reply:
Done, Thank you.